libs/components/src/accordion/vl-accordion.component.ts

```diff
@@ -23,6 +23,7 @@ declare const vl: any;
  * @property {PADDINGS} data-vl-content-padding - De grootte van de padding van de content. Deze padding wordt toegepast op zowel desktop als mobile.
  * @property {boolean} data-vl-bold - Beeldt de toggle-text van de accordion af in bold.
  * @property {boolean} data-vl-disabled - Schakelt het openen en het sluiten van de accordion uit.
+ * @property {boolean} data-vl-is-open - Opent de accordion by default.
```

Comment:
Beetje angelsaksisch... suggesties welkom

Reply:
"standaard"? klikt ook zo raar

Reply:
als ik vlug eens ctrl+f doe in onze app gebruiken we wel vaker `standaard` dan `by default` dus mss beter dat? :)

Reply:
Ik pas het aan